scripts/get-test-pass-rate.js

```diff
@@ -0,0 +1,56 @@
+// This script is used in the "runAfterAll" stage in our E2E test pipeline. It's used 
+// to get timeline and metrics data so that the test pass rate can be calculated in a later step.
+
+const BUILD_ID = process.env.BUILD_ID;
+const ADO_API_TOKEN = process.env.ADO_API_TOKEN;
+const TEST_WORKSPACE = process.env.TEST_WORKSPACE;
+const BUILD_SOURCES_DIRECTORY = process.env.BUILD_SOURCES_DIRECTORY;
+console.log(BUILD_ID);
+console.log(TEST_WORKSPACE);
+// Create output folder - Note: This requires Node.js fs module
+const fs = require('fs');
```

Comment:
These two lines should probably go right above where we use `fs`; here the comment feels confusing because the next thing we do is an http request, not creating a folder. Or maybe bring the folder creating up before doing the REST API call. But let's have a readable sequence of steps, instead of interleaving them.